omnigibson/prims/rigid_prim.py

```diff
@@ -101,8 +101,8 @@ def _post_load(self):
         self.update_meshes()
 
         # Possibly set the mass / density
-        if len(self._collision_meshes) == 0:
-            # We have no collision meshes, so set a negligible mass
+        if len(self._collision_meshes) == 0 and len(self._visual_meshes) > 0:
```

Comment:
when `len(self._collision_meshes) == 0 and len(self._visual_meshes) == 0)`, what should we do?

Reply:
in any case, let's go over all the special casing for mass/density together to make sure it's rock solid.

Reply:
`if len(self._collision_meshes) == 0`, don't apply mass API?